components/blocks/CardGroup.vue

```diff
@@ -11,7 +11,7 @@
         <template #header>
           <span v-if="item.group" class="radius-0 tag is-link has-text-light"
             ><span class="icon"><i class="mdi mdi-account-multiple" /></span
-            ><abbr :title="item.group | expandAcronym">
+            ><abbr :title="expandAcronym(item.group)">
```

Comment:
I don't think group (or any of our tags) are actually abbreviations/acronyms (and definitely not ones we have the full titles for), so I'd remove the `<abbr>` element.